app/utils/estimate_prioritization/estimate_prioritization_test.py

```diff
@@ -0,0 +1,396 @@
+    # Adding these hardcoded values for now to make tests pass
+    sample_df['sampling_start_date'] = datetime.datetime.now()
+    sample_df['sampling_end_date'] = datetime.datetime.now()
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['subgroup_var'] != 'Primary Estimate') & \
+                                (sample_df['study_name'] == 'Study 1'),
+                                mode = 'analysis_static',
+                                test_name = 'ANALYSIS ADJUSTMENT SELECTION',
+                                IDENTIFIER = 1)
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['subgroup_var'] != 'Primary Estimate') & \
+                                (sample_df['study_name'] == 'Study 1'),
+                                mode = 'dashboard',
+                                test_name = 'DASHBOARD ADJUSTMENT SELECTION',
+                                IDENTIFIER = 4)
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['study_name'] == 'Study 2'),
+                                mode = 'analysis_static',
+                                IDENTIFIER = 1,
+                                denominator_value = 300,
+                                test_name = 'POOLING OF IDENTICAL ESTIMATES',
+                                serum_pos_prevalence = 0.3)
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['study_name'] == 'isotype'),
+                                mode = 'analysis_static',
+                                IDENTIFIER = 0,
+                                test_name = 'ISOTYPE')
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['study_name'] == 'isotype comb'),
+                                mode = 'analysis_static',
+                                IDENTIFIER = 0,
+                                test_name = 'ISOTYPE COMBINATION')
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['study_name'] == 'test type'),
+                                mode = 'analysis_static',
+                                IDENTIFIER = 0,
+                                test_name = 'TEST TYPE')
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['study_name'] == 'specimen type'),
+                                mode = 'analysis_static',
+                                IDENTIFIER = 1,
+                                test_name = 'SPECIMEN TYPE')
+
+    estimate_prioritization_test(sample_df,
+                                (sample_df['study_name'] == 'dynamic adjustment'),
+                                mode = 'analysis_dynamic',
+                                IDENTIFIER = 0,
+                                test_name = 'DYNAMIC ADJUSTMENT')
+
+# Test with real estimates queried via our endpoint
+def test_real_estimates():
```

Comment:
appreciate the download of real estimates to properly test this - thanks! Think this makes a lot of sense as a way to check the overall architecture. We're not testing individual functions here, but I think that's fine - to come up with a way to test those would be laborious and prone to breaking